### DISCUSSION TITLE
How migrate ImageProcessor Demo to an Angular project

### DISCUSSION THREAD

**Discussion by dev974-hub** [clientIO/joint#2694] [Q&A]
### Introduction

I'm new to using JointJS and I'm trying to run the ImageProcessor demo in an Angular project.

I followed the tutorial to integate JointJs to angular , but dit not succeed to do with ImageProcessor. 

Is the there any tutorial/ recommandation/ processs ?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
There is the [_Angular_ tutorial](https://docs.jointjs.com/learn/integration/angular).
There are other examples written in _Angular_: https://www.jointjs.com/demos?tags=Angular

In short, the easiest way to do this is to wrap the entire **ImageProcessor** in a single _Angular_ component, following the principles outlined in the tutorial.

**Comment by dev974-hub** [comment]
Thank you so much for your help.

With your help i succeed to integrate the ImageProcessor demo.

But when starting the app i ve got the following error  ' Failed to construct 'Worker': Script at '[file:///C:/Users/sanbox/src/app/jointJs/cv/cv-worker.js] cannot be accessed from origin 'http://localhost:4201' 

do you have any idea on the resolution ?

Best Regards

**Comment by Geliogabalus** [comment]
Which version of Angular you are using? If you are still using webpack Angular builder the issue can be connected with this issue probably: https://github.com/webpack/webpack/issues/16696.

**Comment by dev974-hub** [comment]
Hi ,
For my poc i'm using Angular 14 with all default option used when using Angular CLi.
Here is my poc project: https://github.com/dev974-hub/sandbox-jointjs
I have succeed to setup workers for openCV-js.
All is compiling well with no error but when i drop a task , the task is not rendered correcly :( 
 
Best Regards
